pandas/io/tests/test_packers.py

```diff
@@ -543,25 +543,6 @@ def test_sparse_frame(self):
         self._check_roundtrip(ss3, tm.assert_frame_equal,
                               check_frame_type=True)
 
-    def test_sparse_panel(self):
-
```

Comment:
well if u have a class that doesn't exist and you try to I pickle it will fail

now I don't know if these are actually saved in pickles (maybe not) so may be moot